views/js/runner/runner.js

```diff
@@ -755,9 +755,16 @@ define([
             }
         });
 
-        runner.on('move', function move(type){
-            this.trigger.apply(this, [type].concat([].slice.call(arguments, 1)));
-        });
+        runner
+            .on('move', function move(type){
+                this.trigger.apply(this, [type].concat([].slice.call(arguments, 1)));
+            })
+            .after('destroy', function destroyCleanUp(){
+                areaBroker = null;
+                proxy = null;
+                probeOverseer = null;
+                testStore = null;
```

Comment:
Could you please add also the clean up of the `dataHolder` : `dataHolder.clear()`.